src/backend/distributed/commands/alter_table.c

```diff
@@ -1662,6 +1774,20 @@ ReplaceTable(Oid sourceId, Oid targetId, List *justBeforeDropCommands,
 					 quote_qualified_identifier(schemaName, targetName),
 					 quote_identifier(sourceName));
 	ExecuteQueryViaSPI(query->data, SPI_OK_UTILITY);
+
+	char *coordinatorCommand = NULL;
+	foreach_ptr(coordinatorCommand, coordinatorCommandsToRenameIdentites)
+	{
+		ExecuteQueryViaSPI(coordinatorCommand, SPI_OK_UTILITY);
+	}
+
+	char *workerCommand = NULL;
+	foreach_ptr(workerCommand, workerCommandsToRenameIdentites)
+	{
+		SendCommandToWorkersWithMetadata(workerCommand);
+	}
+
+	MarkIdentitiesAsDistributed(targetId);
```

Comment:
Would be good to add a sentence on why we do this.

Reply:
@gokhangulbiz again, here we are looking for the reason why the identities have to be marked as distributed, even if the table is not distributed.,
And the reason is that, here we may be undistributing the table, which means the table has been distributed before, which means that these identity sequence _exist_ in the workers. Given that they are not deleted from the workers, to maintain consistency and avoid future problems if we redistribute the table, we want to apply all changes happening to the identity sequence in the coordinator to their corresponding sequences in the workers as well.
So, a comment summarizing this reason would be useful :D